packages/master-detail-layout/src/vaadin-master-detail-layout-detail.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright (c) 2025 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css, html, LitElement } from 'lit';
+import { ElementMixin } from '@vaadin/component-base';
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+
+/**
+ * A wrapper element for the detail content, to be used inside a
+ * `<vaadin-master-detail-layout>`. Any content put inside this element will
+ * automatically be shown in the detail area of the layout. If there is no
+ * content, the detail area is hidden. In addition, whenever the contents
+ * of this element change, the layout will automatically start a view transition
+ * to animate between the old and new content.
+ */
+class MasterDetailLayoutDetail extends ElementMixin(LitElement) {
```

Comment:
We don't need `ElementMixin` here since it's mainly for usage statistics and it's enough to apply it to the main component. Also it applies `DirMixin` which isn't needed here anyway due to `display: contents`.